app/logic/userInsertFunctions.py

```diff
@@ -127,6 +128,7 @@ def updateSupervisorRecord(supervisor):
     supervisor.CPO = tracyUser.CPO
     supervisor.ORG = tracyUser.ORG
     supervisor.DEPT_NAME = tracyUser.DEPT_NAME
+    supervisor.isActive = bool(tracyUser)
```

Comment:
This should be set to `True`, as it only gets to this point when tracyUser is valid. Using `bool(tracyUser)` implies that it could potentially be false, which is incorrect.

Reply:
Fixed